python/Scripts/translateshader.py

```diff
@@ -85,7 +89,10 @@ def main():
         
     # Bake translated materials to flat textures.
     baseType = mx_render.BaseType.FLOAT if opts.hdr else mx_render.BaseType.UINT8
-    baker = mx_render_glsl.TextureBaker.create(bakeWidth, bakeHeight, baseType)
+    if platform == "darwin":
```

Comment:
This has the same issue as baketexture.py naturally as the MSL baker is only ever used on Mac.